de.dlr.sc.virsat.model.concept.test/src/de/dlr/sc/virsat/model/concept/generator/beans/GenerateCategoryBeansTest.xtend

```diff
@@ -28,7 +29,7 @@ import org.junit.runner.RunWith
 
 @RunWith(XtextRunner)
 @InjectWith(ConceptLanguageTestInjectorProvider)
-class GenerateCategoryBeansTest {
+class GenerateCategoryBeansTest extends AExternalModelTest {

```

Comment:
I think i would go for a utility class as well instead of inheritance. Or maybe it is something that can be placed to AConceptTestCase. But there we have to be super careful that we are not creating havoc on the plugin dependencies etc.